monitors/index.js

```diff
@@ -119,31 +153,46 @@ async function run(address, shouldPoll, botMonitorObject, walletMonitorObject, p
 const Poll = async function(callback) {
   try {
     if (!process.env.EMP_ADDRESS) {
-      throw new Error(
-        "Bad environment variables! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party."
-      );
+      throw "Bad environment variables! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party.";
     }
 
     if (!process.env.BOT_MONITOR_OBJECT || !process.env.WALLET_MONITOR_OBJECT) {
-      throw new Error("Bad input arg! Specify a `BOT_MONITOR_OBJECT` & `WALLET_MONITOR_OBJECT` to track.");
+      throw "Bad input arg! Specify a `BOT_MONITOR_OBJECT` & `WALLET_MONITOR_OBJECT` to track.";
     }
 
-    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 10_000;
+    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 10000;
 
     // Bot objects to monitor. For each bot specify a name, address and the thresholds to monitor.
     const botMonitorObject = JSON.parse(process.env.BOT_MONITOR_OBJECT);
 
     // Wallet objects to monitor.
     const walletMonitorObject = JSON.parse(process.env.WALLET_MONITOR_OBJECT);
 
-    if (!process.env.PRICE_FEED_CONFIG) {
-      throw new Error("Bad input arg! Specify `PRICE_FEED_CONFIG` to define the price feed settings");
+    if (!process.env.UNISWAP_PRICE_FEED_CONFIG || !process.env.MEDIANIZER_PRICE_FEED_CONFIG) {
+      throw "Bad input arg! Specify `PRICE_FEED_CONFIG` and `MEDIANIZER_PRICE_FEED_CONFIG` to define the price feed settings.";
```

Comment:
Do we have a preference between throwing a string vs an `Error` object? Are there reasons to prefer one over the other?

cc @nicholaspai

Reply:
My default has been to throw explicitly an `Error` object as its more explicit FYI

Reply:
In this [PR](https://github.com/UMAprotocol/protocol/pull/1450) I refactored the bots to how a string, not an `Error` object. We should have a convention for this.